deployments/examples/ocis_wopi/docker-compose.yml

```diff
@@ -209,6 +214,13 @@ services:
     logging:
       driver: "local"
     restart: always
+    healthcheck:
```

Comment:
Does it make sense to use the `healthcheck` at all? Does `depends_on` respect `healthcheck`?
If it works, we also should add it for the onlyoffice.